src/components/Picker/index.android.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I'm not sure I understand why we need three different, largely identical files for the picker.

Reply:
is the style the only change? could we have that an input for another compnent perharphs? No real idea, just wandering

Reply:
Yeah, the styles are basically the only difference. I like the idea of just exporting the proper styles from the platform-specific files and DRYing this up more. I'll be working on that today.